src/content/en/fundamentals/performance/rail.md

```diff
@@ -1,171 +1,350 @@
     </tr>
     <tr>
-      <td data-th="Delay">10,000+ms</td>
-      <td data-th="User Reaction">The user is frustrated and is likely to abandon the task; they may or may not come back later.</td>
+      <td data-th="Delay">10000ms or more</td>
+      <td data-th="User Reaction">Beyond 10000 milliseconds (10 seconds), users are frustrated
+      and are likely to abandon tasks. They may or may not come back later.</td>
     </tr>
   </tbody>
 </table>
 
-## Response: respond in under 100ms {: #response }
+Users perceive performance delays differently, depending on network conditions and hardware.
+For example, loading an experience in 1000ms is plausible on a powerful desktop machine over a
+fast Wi-Fi connection, so users have grown accustomed to a 1000ms loading experience. But for
+mobile devices over slow 3G connections, loading in 5000ms is a more realistic goal, so mobile
+users are generally more patient.
 
-You have 100ms to respond to user input before they notice a lag.
-This applies to most inputs, such as clicking buttons, toggling form
-controls, or starting animations. This does not apply to touch drags or
-scrolls.
+## Response: respond in under 50ms {: #response }
 
-If you don't respond, the connection between action and reaction is broken. Users will notice.
+**Goal**: Complete a transition initiated by user input within 100ms. Users spend the majority
+of their time waiting for sites to respond to their input, not waiting for the sites to load.
 
-While it may seem obvious to respond to user's actions immediately,
-that's not always the right call.
-Use this 100ms window to do other expensive work, but be careful not to block the user.
-If possible, do work in the background.
+**Guidelines**:
 
-For actions that take longer than 500ms to complete, always provide feedback.
+* Respond to user input within 50ms, or else the connection between action and
+  reaction is broken. This applies to most inputs, such as clicking buttons, toggling form
+  controls, or starting animations. This does not apply to touch drags or scrolls.
+* Though it may sound counterintuitive, it's not always the right call to respond to user
+  input immediately. You can use this 100ms window to do other expensive work. But be
+  careful not to block the user. If possible, do work in the background.
+* For actions that take longer than 500ms to complete, always provide feedback.
 
 ## Animation: produce a frame in 10ms {: #animation }
 
-Animations aren't just fancy UI effects. For example, scrolling and touch
-drags are types of animations.
+**Goal**: Produce frames as fast as the user's hardware can display them. Users notice when
+animation frame rates vary.
+
+**Guidelines**:
+
+* Produce each frame in an animation in 10ms or less. Technically, the maximum budget for
+  each frame is 16ms (1000ms / 60 frames per second ≈ 16ms), but browsers need about 6ms to
+  render each frame, hence the guideline of 10ms per frame.
```

Comment:
Isn't this tied to human perception? I.e. we perceive something as smooth if it's 60FPS. So shouldn't this be the goal?

Reply:
Thinking back through this, you're right to call out that we don't tie this back to human perception right now. I think optimizing for that and ensuring smoothness is pretty valid for the goal. Did you have additional edits you'd like to make here?

Reply:
I'm going to move "Produce each frame in 10ms..." up to the Goals list